dataprofiler/tests/profilers/profiler_options/test_unstructured_text_profile_options.py

```diff
@@ -127,6 +153,18 @@ def test_validate(self):
                      "TextProfilerOptions.stop_words must be None "
                      "or list of strings."
                  ]),
+            dict(prop='top_k_words',
+                 value_list=['a', [1, 2], ['a', 1, 'a']],
+                 errors=[
+                     "TextProfilerOptions.top_k_words "
+                     "must be None or integer."
+                 ]),
+            dict(prop='top_k_chars',
+                 value_list=['a', [1, 2], ['a', 1, 'a']],
+                 errors=[
+                     "TextProfilerOptions.top_k_chars "
+                     "must be None or integer."
+                 ]),
```

Comment:
What if they set it to be negative? We should cover this case.

Reply:
added